### ISSUE TITLE
Do not score checked in libraries

### ISSUE THREAD

**Issue by mihaibivol** [rosedu/challenge#56]
Add a method for not scoring commits adding 3rd party code such as *.min.js and *.min.css

As I am not familiar with the codebase I can't actually propose an accurate solution.

**Comment by marianitadn** [comment]
You can't actually know. But the mentor can add a smaller rating to the PR, and lines added have less value than removed lines. So the score will be low.

**Comment by mihaibivol** [comment]
Is this documented somewhere?

**Comment by marianitadn** [comment]
No. We talked and we might have found a solution. Will let you know.

**Comment by mihaibivol** [comment]
Fun fact: http://githut.info/
Probably the same cause of js being ahead of everything else
